src/resolvers/nodeResolver.ts

```diff
@@ -27,7 +27,7 @@ const versionCache: {
 			.get(1)
 			.then((res: { results: Array<{ name: string }>; next?: string }) => {
 				const curr = _(res.results).map('name').filter(versionTest).value();
-				const tags = prev.concat(curr);
+				const tags = prev.concat(curr as string[]);
```

Comment:
Ah, I probably wrote it when 2.3 was the latest, and then 2.4 got stricter with some of these things